runtime/vm/jvminit.c

```diff
@@ -3761,6 +3761,32 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 	}
 #endif /* defined(OMR_GC_COMPRESSED_POINTERS) && defined(OMR_GC_FULL_POINTERS) */
 
+#if JAVA_SPEC_VERSION >= 18
+	{
+		/* JEP 421: Deprecate Finalization for Removal
+		 * Expecting values: "enabled" and "disabled". The default value for now will be "enabled", i.e., finalization is enabled by default.
+		 * Other values will cause an error, additional values may be added in the future.
+		 */
```

Comment:
Please fold this comment so it is complete visible here.

Reply:
Rebased w/ https://github.com/eclipse-openj9/openj9/commit/86231f8ec44a97115ece370e70ce405e1ab4241c
Adjusted comments along with following code block.